contracts/MemoworldNFTMint01.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.6 <0.9.0;
+pragma abicoder v2;
+
+import {IMemoworldNFT} from './interfaces/IMemoworldNFT.sol';
+
+contract MemoworldNFTMint01 {
+  // Address representating ETH (e.g. in payment routing paths)
+  address public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+
+  // Indicates that this plugin requires delegate call
+  bool public immutable delegate = true;
+
+  event RouteMint(address indexed sender, uint256 indexed tokenId, uint256 indexed amount, string data);
+
+  constructor() {}
+
+  function execute(
+    address[] calldata path,
+    uint256[] calldata amounts,
+    address[] calldata addresses,
+    string[] calldata data
+  ) external payable returns (bool) {
+    IMemoworldNFT memoContract = IMemoworldNFT(addresses[addresses.length - 1]);
+
+    memoContract.mint{value: amounts[1]}(addresses[0], amounts[5], amounts[6], data[0]);
```

Comment:
what information are you transporting through data[0]?

Reply:
The data[0] is just pass to a parent function "_mint". It seem just a remark for mint. I can set it a static empty string instead.